examples/transmitter/swagger_server/business_logic/__init__.py

```diff
@@ -117,8 +113,10 @@ def verification_request(state: Optional[str], client_id: str) -> None:
     )
     stream.process_SET(security_event)
 
-def _well_known_sse_configuration_get(url_root: str, 
-                                      issuer: Optional[str] = None) -> TransmitterConfiguration:
+
+def _well_known_sse_configuration_get(url_root: str,
+                                      issuer: Optional[str] =
+                                      None) -> TransmitterConfiguration:
```

Comment:
I appreciate you trying to bring this within proper line length, but breaking on that equals sign is hard to read. What about this alternative?
```
def _well_known_sse_configuration_get(
    url_root: str,
    issuer: Optional[str] = None
) -> TransmitterConfiguration:
```

Reply:
True! Will change this.